llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -2515,6 +2525,10 @@ bool SPIRVInstructionSelector::selectIntrinsic(Register ResVReg,
     return selectExtInst(ResVReg, ResType, I, CL::rsqrt, GL::InverseSqrt);
   case Intrinsic::spv_sign:
     return selectSign(ResVReg, ResType, I);
+  case Intrinsic::spv_firstbituhigh:
```

Comment:
Maybe add a comment saying we are only setting the GL extension because there is no CL equivalent.